### ISSUE TITLE
[BUG] attempt to index field 'Cfg' (a nil value)

### ISSUE THREAD

**Issue by lcpichette** [nvim-telekasten/telekasten.nvim#349]
**Please confirm**
- [x] I am running the latest version of this plugin

**Describe the bug**

For every Telekasten command I run I get an error about Cfg indexing. Even just running `:Telekasten` gives me the error.

**To Reproduce**
1. Start nvim
2. Add Telekasten to Astrovim via lazyvim `/lua/plugins/telekasten.lua` => `return ...provided lazyvim code`
3. Close and reopen neovim
4. Run any telekasten command
5. See error

**Expected behavior**

**Screenshots**

![2024-10-17@14 58 32 October-17@2x](https://github.com/user-attachments/assets/e367482f-cdcb-4aa5-a672-6af736e993c3)

**Operating system (please complete the following information):**
 - OS: MacOS 15.1 on M1

**Additional context**

This is my first time using this plugin, so I've likely just installed something incorrectly. But I've installed other neovim plugins via lazyvim and astrovim before, and have never encountered this error.

**Comment by lambtho12** [comment]
Is it still happening ? I do not have any issue on my side.

**Comment by lcpichette** [comment]
Just tried again and I get the same error, but I didn't disable all of my other plugins like I had before. See PR description for reproducible steps and system information. If you performed the reproducible steps with my same system information then I'd make sure that you're using AstroVim as I specified in the PR description.

Thank you for taking a look.

![2024-10-26@21 29 03 October-26@2x](https://github.com/user-attachments/assets/ca5e5b5b-0a60-4023-9184-b09b7158596e)

**Comment by Docfips** [comment]
Using LunarVim, same issue.

**Comment by mediapathic** [comment]
Same issue here after an update of Astronvim (which broke a bunch of other stuff and I'm starting to suspect a trend there)

**Comment by handdara** [comment]
I also am having this issue. Redoing my nvim config from scratch at the moment and had the same issue with this.

**Comment by lambtho12** [comment]
> EDIT: Well... i feel dumb, but I didn't save the file that called `require('telekasten').setup{...}` function, so it just was never getting called. Probably something like that for you too.

So, once you called the setup function properly everything started to work ? Is that right ?

@lcpichette , @mediapathic , @Docfips could you check ? I am not able to reproduce...

**Comment by lcpichette** [comment]
@lambtho12 Yes I do have the setup file. See step 2.

**Comment by mediapathic** [comment]
In my case, it actually did turn out to be the setup function; the Astronvim update messed up such that the file containing that was not actually being called. I apologize for the noise.

**Comment by lcpichette** [comment]
I've found `zk` which does what I want it to: https://zk-org.github.io/zk/config/config-group.html It also has its own neovim plugin https://github.com/zk-org/zk-nvim. Not to say it's better or worse then Telekasten, I just didn't face any odd bugs when trying to use it. 

It sounds like others who had the same issue as me resolved it with some config tweaks -- so this is likely a PEBCAK issue. Closing this issue under that assumption.
